components/x-gift-article/src/GiftArticle.scss

```diff
@@ -50,12 +50,6 @@
 	);
 }
 
-.x-gift-article__enterprise-label {
-	background-color: oColorsByName('black-10');
-	color: oColorsByName('black');
-	margin: 0px 6px;
-}
```

Comment:
Thought: Can this change be related to the bug Fabio reported?

Reply:
Not quite! This one's [to remove an old label](https://financialtimes.atlassian.net/jira/software/projects/ENTST/boards/1569?selectedIssue=ENTST-152). I've added additional styles to fix the styling bug 😄